### PR TITLE
WindowTextExtractor: Add version 1.15.0

### PR DESCRIPTION
**Pull request by AlexanderPro**
WindowTextExtractor is an absolutely free and simple in use system tool to:
* Extract text from modal windows, buttons, textboxes, lists, etc
* Show passwords stored behind asterisks (*****) from most of the windows apps
* Show detailed window and process information 
* Show process environment variables
* Show or hide almost any desktop window
* Take a window screenshot
* Record window stream in avi file
* OCR support (including text, bar codes and QR codes) for Windows 10, 11

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by rasa — APPROVED** [review]

File: bucket/windowtextextractor.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.15.0",
+    "description": "A tool allows you to get a text from any window of an operating system including asterisk passwords.",
+    "homepage": "https://github.com/AlexanderPro/WindowTextExtractor",
+    "license": "MIT",
+    "url": "https://github.com/AlexanderPro/WindowTextExtractor/releases/download/v1.15.0/WindowTextExtractor_v1.15.0.zip",
+    "hash": "454f477dd62a2092a2aa09e751bc6a70cdb847a25d060b7e57bf7b8a426ba3c6",
+    "shortcuts": [
```
Comment (HUMORCE) [inline bucket/windowtextextractor.json:8]:
```suggestion
    "hash": "454f477dd62a2092a2aa09e751bc6a70cdb847a25d060b7e57bf7b8a426ba3c6",
    "pre_install": [
        "if ($architecture -eq '64bit') {",
        "    Rename-Item \"$dir\\WindowTextExtractor.exe\" 'WindowTextExtractor32.exe'",
        "    Rename-Item \"$dir\\WindowTextExtractor64.exe\" 'WindowTextExtractor.exe'",
        "}"
    ],
    "shortcuts": [
```

**Review by HUMORCE — CHANGES_REQUESTED** [review]
64bit

File: bucket/windowtextextractor.json (hunk 1 above)
Comment (AlexanderPro) [reply bucket/windowtextextractor.json:8]: WindowTextExtractor.exe is for 32 bit and 64 bit OS. WindowTextExtractor64.exe is a special file and not for a manual run.

Comment (HUMORCE) [reply bucket/windowtextextractor.json:8]: sorry, I did not run it for check.

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**